docs/intro/tutorial03.txt

```diff
@@ -449,6 +449,7 @@ file, go ahead and add an ``app_name`` to set the application namespace:
     :filename: polls/urls.py
 
     from django.conf.urls import url
+    from . import views
```

Comment:
If you want to try amending this pull request, you could add a newline above this line for consistency and amend the commit message to "Added a missing import to tutorial 3." If not, it's easy enough to make these updates while merging. Thanks!